packages/connection/src/Manager.php

```diff
@@ -28,6 +29,155 @@ class Manager implements Manager_Interface {
+
+	/**
+	 * Creates an instance of the connection manager.
+	 */
+	public function __construct() {
+	}
+
+	/**
+	 * Initializes required listeners. This is done separately from the constructors
+	 * because some objects sometimes need to instantiate separate objects of this class.
+	 */
+	public function init() {
+
+		$is_jetpack_xmlrpc_request = $this->setup_xmlrpc_handlers(
+			$_GET,
+			$this->is_active(),
+			$this->verify_xml_rpc_signature()
+		);
+
+		// All the XMLRPC functionality has been moved into setup_xmlrpc_handlers.
+		if (
+			! $is_jetpack_xmlrpc_request
+			&& is_admin()
+			&& isset( $_POST['action'] ) // phpcs:ignore WordPress.Security.NonceVerification
+			&& (
+				'jetpack_upload_file' === $_POST['action']  // phpcs:ignore WordPress.Security.NonceVerification
+				|| 'jetpack_update_file' === $_POST['action']  // phpcs:ignore WordPress.Security.NonceVerification
+			)
+		) {
+			$this->require_jetpack_authentication();
+			$this->add_remote_request_handlers();
+		} else {
+			if ( $this->is_active() ) {
+				add_action( 'login_form_jetpack_json_api_authorization', array( &$this, 'login_form_json_api_authorization' ) );
+				add_filter( 'xmlrpc_methods', array( $this, 'public_xmlrpc_methods' ) );
+			} else {
+				add_action( 'rest_api_init', array( $this, 'initialize_rest_api_registration_connector' ) );
+			}
+		}
+	}
+
+	/**
+	 * Sets up the XMLRPC request handlers.
+	 *
+	 * @param Array                  $request_params incoming request parameters.
+	 * @param Boolean                $is_active whether the connection is currently active.
+	 * @param Boolean                $is_signed whether the signature check has been successful.
+	 * @param \Jetpack_XMLRPC_Server $xmlrpc_server (optional) an instance of the server to use instead of instantiating a new one.
+	 */
+	public function setup_xmlrpc_handlers(
+		$request_params,
+		$is_active,
+		$is_signed,
+		\Jetpack_XMLRPC_Server $xmlrpc_server = null
+	) {
+		if (
+			! isset( $request_params['for'] )
+			|| 'jetpack' !== $request_params['for']
+		) {
+			return false;
+		}
+
+		// Alternate XML-RPC, via ?for=jetpack&jetpack=comms.
+		if (
+			isset( $request_params['jetpack'] )
+			&& 'comms' === $request_params['jetpack']
+		) {
+			if ( ! Constants::is_defined( 'XMLRPC_REQUEST' ) ) {
+				// Use the real constant here for WordPress' sake.
+				define( 'XMLRPC_REQUEST', true );
+			}
+
+			add_action( 'template_redirect', array( $this, 'alternate_xmlrpc' ) );
+
+			add_filter( 'xmlrpc_methods', array( $this, 'remove_non_jetpack_xmlrpc_methods' ), 1000 );
+		}
+
+		if ( ! Constants::get_constant( 'XMLRPC_REQUEST' ) ) {
+			return false;
+		}
+		// Display errors can cause the XML to be not well formed.
+		@ini_set( 'display_errors', false ); // phpcs:ignore
+
+		if ( $xmlrpc_server ) {
+			$this->xmlrpc_server = $xmlrpc_server;
+		} else {
+			require_once JETPACK__PLUGIN_DIR . 'class.jetpack-xmlrpc-server.php';
+			$this->xmlrpc_server = new Jetpack_XMLRPC_Server();
```

Comment:
I was able to make this work by telling Composer to look for it in the global namespace:
```suggestion
			$this->xmlrpc_server = new \Jetpack_XMLRPC_Server();
```